src/assets.test.ts

```diff
@@ -32,8 +32,6 @@ describe('assets', () => {
       'mcUSDxOLD.png',
       'mcREAL.png',
       'stabilUSD.png',
-      'USDC.png',
-      'USDT.png',
```

Comment:
why delete?

Reply:
These were exceptions for the asset size. Not needed anymore.